scripts/game_structure/load_cat.py

```diff
@@ -121,6 +121,8 @@ def json_load():
             new_cat.adoptive_parents = cat["adoptive_parents"] if "adoptive_parents" in cat else []
             
             new_cat.genderalign = cat["gender_align"]
+            # new_cat.pronouns = cat["pronouns"]
+            new_cat.pronouns = cat["pronouns"] if "pronouns" in cat else [new_cat.default_pronouns[0].copy()]
```

Comment:
Note -- cats from old saves will be assigned they/them pronouns irrregardless of the setting. This is probably fine, but important to note.